operator/v1/types.go

```diff
@@ -284,6 +297,10 @@ type KlusterletSpec struct {
 	// DeployOption contains the options of deploying a klusterlet
 	// +optional
 	DeployOption KlusterletDeployOption `json:"deployOption,omitempty"`
+
+	// RegistrationConfiguration contains the configuration of registration
+	// +optional
+	RegistrationConfiguration RegistrationConfiguration `json:"registrationConfiguration,omitempty"`
```

Comment:
omitempty doesn take effect on struct, consider make this a pointer

Reply:
👌🏻